src/index.ts

```diff
@@ -120,12 +122,12 @@ function view(state, actions) {
           actions.onEditorInput,
           actions.onEditorClose,
         )
-        : null,
+        : Empty(),
     ]
   )
 }
 
-function registerEventHandlers(el, actions) {
+function registerEventHandlers(el: El, actions: Actions) {
```

Comment:
A comment should be added here to describe this function, similar to render() and _renderNodes().